GTG/gtk/browser/main_window.py

```diff
@@ -260,6 +261,17 @@ def _init_widget_aliases(self):
 
         self.tagpopup = TagContextMenu(self.req, self.app)
 
+        # Setup search syntax popup button
+        self.search_entry.set_icon_from_icon_name(Gtk.EntryIconPosition.SECONDARY, 'help-faq-symbolic')
```

Comment:
Couldn't we do this directly in the Builder XML?

Reply:
huh, didn't check but it turns out it can be done